relay_test.go

```diff
@@ -490,6 +490,29 @@ func TestRelayConnection(t *testing.T) {
 	assert.Contains(t, err.Error(), errTest.Error(), "Unexpected error")
 }
 
+func TestSilentDrop(t *testing.T) {
+	getHost := func(call relay.CallFrame, conn relay.Conn) (relay.Peer, error) {
+		return relay.Peer{}, relay.RateLimitDropError{}
+	}
+
+	// Note: we cannot use WithTestServer since we override the RelayHosts.
+	opts := testutils.NewOpts().
+		SetServiceName("relay").
+		SetRelayHosts(hostsFunc(getHost)).
+		SetProcessPrefixes("nod", "nodejs-hyperbahn", "", "hyperbahn")
+	relay := testutils.NewServer(t, opts)
+	defer relay.Close()
+
+	// Create a client that is listening so we can set the expected host:port.
+	clientOpts := testutils.NewOpts().SetProcessName("nodejs-hyperbahn")
```

Comment:
don't need to set process prefixes or process names in this test, it's unrelated to silent drop